pkg/model/components/context.go

```diff
@@ -63,3 +65,35 @@ func (c *OptionsContext) UsesKubenet() (bool, error) {
 		return false, fmt.Errorf("No networking mode set")
 	}
 }
+
```

Comment:
I agree.   It might end up in its own package...

I really want to remove everything from the templates though, and this is another step in that direction.